packages/core/src/node/env-variables/env-variables-server.ts

```diff
@@ -53,24 +52,26 @@ export class EnvVariablesServerImpl implements EnvVariablesServer {
         return this.envs[key];
     }
 
-    async getUserHomeFolderPath(): Promise<string> {
-        return os.homedir();
+    async getUserHomeFolder(): Promise<string> {
+        return new URI(os.homedir()).toString();
```

Comment:
You have to do `FileUri.create(os.homedir()).toString()`. Same for the others.

Reply:
ok

Reply:
Done